src/Raven.Server/Rachis/FollowerAmbassador.cs

```diff
@@ -804,7 +807,15 @@ private long InitialNegotiationWithFollower()
                 } while (true);
             }
         }
-        
+
+        private int GetFollowerVersion(LogLengthNegotiationResponse llr)
+        {
+            var version = llr.CommandsVersion ?? 400_000;
```

Comment:
I think that 40_000 would be enough

Reply:
Let's be on the (very) safe side this time :)